site/pages/retirements/[beneficiary]/[retirement_index].tsx

```diff
@@ -4,18 +4,15 @@ import { ParsedUrlQuery } from "querystring";
 
 import { urls } from "@klimadao/lib/constants";
 import { KlimaRetire as IKlimaRetire } from "@klimadao/lib/types/subgraph";
-import {
-  getRetirementDetails,
-  queryKlimaRetireByIndex,
-} from "@klimadao/lib/utils";
-
+import { getRetirementDetails } from "@klimadao/lib/utils";
 import { getPledgeByAddress } from "components/pages/Pledge/lib/firebase";
 import { Pledge } from "components/pages/Pledge/types";
 import { SingleRetirementPage } from "components/pages/Retirements/SingleRetirement";
 import { getAddressByDomain } from "lib/getAddressByDomain";
 import { getIsDomainInURL } from "lib/getIsDomainInURL";
 import { loadTranslation } from "lib/i18n";
 import { INFURA_ID } from "lib/secrets";
+import { queryKlimaRetireByIndex } from "../../../../carbonmark/lib/retirementDataQueries/retirementDataViaPolygonDigitalCarbon";
```

Comment:
This should work right?

```suggestion
import { queryKlimaRetireByIndex } from "lib/retirementDataQueries/retirementDataViaPolygonDigitalCarbon";
```

Reply:
I changed the above as I was receiving errors on the local test build-all. Committed the suggestion here.